maqs-appium/src/test/java/com/cognizantsoftvision/maqs/appium/AppiumConfigUnitTest.java

```diff
@@ -19,7 +18,7 @@
 /**
  * The type Appium config unit test.
  */
-public class AppiumConfigUnitTest extends BaseGenericTest {
```

Comment:
Why extend base Appium when generic works?

Reply:
reverted back to use baseGenericTest